COLLABORATOR_GUIDE.md

```diff
@@ -38,9 +39,9 @@ necessary.
 
 ## Accepting Modifications
 
-All modifications to the Node.js code and documentation should be
-performed via GitHub pull requests, including modifications by
-Collaborators and TC members.
+All modifications to the Node.js on ChakraCore code and documentation
+should be performed via GitHub pull requests, including modifications
+by Collaborators and TC members.
```

Comment:
This is a bug in the upstream. Rather than fix it here, I'll open a PR upstream.